cmd/dumpstdlibast/dumpstdlibast.go

```diff
@@ -33,7 +33,7 @@ func main() {
 		panic(err)
 	}
 
-	node, err := jsonnet.SnippetToAST("<std>", string(buf))
+	node, err := transformations.SnippetToAST("<std>", string(buf))
 	if err != nil {
 		panic(err)
 	}
```

Comment:
Since I can't leave a comment on a line that didn't change, I'll ask as close by as possible: Is it correct that you left the `init` function in the `header` value here for now with the intention to come through in a second pass and remove the duplicated installation of the standard library AST?

Reply:
I intend to do the whole astgen cleanup in separate PR. I like to keep the changes that move stuff around as small and focused as possible.